server/controllers/inventory/depots/index.js

```diff
@@ -234,3 +234,30 @@ function detail(req, res, next) {
     .catch(next)
     .done();
 }
+
+// Get depots by their uuids
+function getDepotGroup(req, res, next) {
+  const { depotUuids } = req.query;
+  const uuidGroup = [].concat(depotUuids);
```

Comment:
You can do this entire thing by added only two lines to the `list()` function.

The first is changing
```js
  const options = db.convert(req.query, ['uuid']);
```
to 
```js
  const options = db.convert(req.query, ['uuid', 'uuids']);
```

The second is adding a line below:
```js
  filters.equals('uuid', 'uuid', 'd');
```
add this line:
```js
  filters.equals('uuid', 'uuid', 'd');
  filters.equals('uuids', 'uuids', 'd', true);
```
Everything should now work as expected.